CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Changed
+- Hosts can authenticate from Google Cloud Function (GCF) using a GCE instance
```

Comment:
is this a new addition that was introduced or something that was changed? If new addition, please create a new "Added" title above the Changed. Like so

## [Unreleased]

### Added
<your change>

### Changed
...